test/verify/check-superuser

```diff
@@ -359,40 +359,34 @@ class TestSuperuserDashboard(MachineCase):
         b.click('#troubleshoot-dialog button:contains("Connect")')
         b.wait_popdown('troubleshoot-dialog')
 
-        # The superuser indicator should be on the Overview page and not in the Shell
+        # The superuser indicator in the Shell should apply to machine2
 
-        b.wait_text("#super-user-indicator", "")
-        b.enter_page("/system", host="10.111.113.2")
-        b.wait_in_text(".ct-overview-header-actions", "Administrative access")
```

Comment:
Is showing the status in the Overview still a thing in some cases? I don't see that PR removing it. Then the test should assert that it's not present?

Reply:
Yes, when a new machine is on the dashboard of an old machine.  And we indeed need for more tests for that.